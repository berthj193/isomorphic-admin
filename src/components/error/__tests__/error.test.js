import React from 'react';
import Error from '../error';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('<Error />', () => {
  const getComponent = (props = {}) => shallowToJson(shallow(<Error {...props} >I am ok :)</Error>));

  it('should display children, if error props is falsy', () =>
    expect(
      getComponent(),
    ).toMatchSnapshot()
  );

  it('should display children, if error props is falsy', () =>
    expect(
      getComponent({ error: 'I am not ok :(' })
    ).toMatchSnapshot()
  );
});
