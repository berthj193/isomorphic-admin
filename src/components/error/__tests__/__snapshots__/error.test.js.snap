// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should display children, if error props is falsy 1`] = `"I am ok :)"`;

exports[`<Error /> should display children, if error props is falsy 2`] = `
<span
  className="error"
>
  I am not ok :(
</span>
`;
